content/en/software_catalog/_index.md

```diff
@@ -60,22 +60,26 @@ Built on real-time telemetry and automated metadata collection, Software Catalog
 {{< callout url="https://www.datadoghq.com/product-preview/internal-developer-portal/" d_target="#signupModal" btn_hidden="false" header="Opt in to the preview for our Internal Developer Portal!" >}}
 {{< /callout >}}
 
-### Use cases
+## Key terminology
 
-#### Governance and optimization
-- Providing engineering leadership with a high-level view of best practices across teams and services through [Service Scorecards][9].
-- Reducing application risks by finding and fixing known security vulnerabilities in the dependencies of your services.
-- Understanding trends and identifying inefficiencies in the costs related to your services.
+Service
+: In APM, a [service][10] is a group of related endpoints, queries, or jobs that perform a piece of work for your application. For example, a service could be:
 
-#### Evaluate monitoring coverage  
-- Detecting which services aren’t reporting observability data or having that data monitored.
-- Facilitating [tagging best practices][6] and checking for recommended setup configurations to optimize [cross-telemetry insights][7].
-- Spotting issues like missing SLOs, monitors, or services without ownership.
+* A group of URL endpoints may be grouped together under an API service.
+* A group of DB queries that are grouped together within one database service.
+* A group of periodic jobs configured in the crond service.
 
-#### Streamline collaboration during incidents
-- Improving the on-call experience for everyone by establishing correct ownership information and communication channels, alongside streamlined access to monitoring and troubleshooting details.
-- Embedding links to solutions and troubleshooting tools such as runbooks and documentation directly in the observability tooling engineers are already using.
-- Speeding incident recovery by increasing confidence and simplifying locating owners of upstream and downstream services and dependencies.
+Through custom instrumentation in APM, you may create an arbitrary `service`. In pratice, microservice-based architecture includes multiple APM services, each measuring the performance of sub-components of the application through [Trace Metrics][2]. 
+
+In Software Catalog, you can collect non-instrumented servicse by declaring them through [metadata][12] or import additional services through external sources like [Backstage][13] or [ServiceNow][14]. 
+
+System
+: In Software Catalog, a system is a group of components that cooperate to perform a broader piece of function. For example, you may group multiple instrumented APM services into a system because they are operated by the same team. You may also use `system` to represent a full microservice-based piece of architecture and include other components like APIs, datastores, queues, and other common building blocks.
```

Comment:
```suggestion
: In the Software Catalog, a system is a group of components that cooperate to perform a broader function. For example, you can group multiple instrumented APM services into a system because they are operated by the same team. You can also use `system` to represent a full microservice-based architecture, and include components like APIs, datastores, queues, and other common building blocks.
```